clang/test/AST/ast-dump-template-decls.cpp

```diff
@@ -222,7 +220,7 @@ int binTempl<int, U>;
 
 template<class U>
 float binTempl<float, U> = 1;
-// CHECK:      VarTemplatePartialSpecializationDecl 0x{{[^ ]*}} <line:{{[0-9]+}}:1, line:{{[0-9]+}}:24> col:7 binTempl 'float' cinit
```

Comment:
Why are we losing 'const-int' (Is that what cinit means?) here?  Is that perhaps important?

Reply:
I removed `cinit` because the output will contain "`explicit_specialization`" before it. 

Despite the output being identical to our current output, this somehow seems to affect whether the tests pass if `--check-prefix=...` is removed from the test command line arguments. If the check prefix arguments are retained and we instead make the expected output lines for `DIRECT`/`SERIALIZED` the same, the tests pass. Merging the two lines into a single `CHECK` line results in the change being necessary for the tests to ass...